libs/utils/utils.js

```diff
@@ -330,3 +342,19 @@ export function updateObj(obj, defaultObj) {
   return obj;
 }
 
+export const throttle = (delay = 250, throttled = () => {}, opts = {}, ...args) => {
```

Comment:
I think we should only be putting these sort of utils into "utils" (terrible name for this file in hindsight) when more than one block uses it.